setup.py

```diff
@@ -23,7 +23,8 @@
         license='MIT license',
         packages=['surfinpy'],
         zip_safe=False,
-        install_requires=['scipy', 'numpy', 'jupyter', 'pymatgen', 'numpy', 'matplotlib', 'coverage', 'coveralls', 'seaborn'],
+        python_requires='>=3.6',
+        install_requires=['scipy', 'numpy', 'jupyter', 'pymatgen', 'numpy', 'matplotlib', 'seaborn','pyyaml'],
```

Comment:
Is jupyter in actual install requirement here (see #22)? Also, you added pandas to list of dependencies in the README, which is indirectly included via seaborn, however shouldn't it be listed explicitly here?

Further the install requirements don't have version constraints, maybe worth adding them if they are in the `requirements.txt` which is not available via an install from PyPI.